docs/about/policies.md

```diff
@@ -0,0 +1,101 @@
+# General Policies v1.0
+
+## Content
+
+- **Scope:** Neurophysiology research. Raw and derived experimental data. Content
+  must not violate privacy or copyright, or breach confidentiality or non-disclosure
+  agreements for data collected from human subjects.
+- **Status of research data:** Any status is accepted, from any stage of the
+  research lifecycle.
+- **Eligible depositors:** Anyone may register as user of DANDI. All users are
+  allowed to deposit content for which they possess the appropriate rights.
+- **Ownership:** By uploading content, no change of ownership is implied and no
+  property rights are transferred to the DANDI team. All uploaded content remains
+  the property of the parties prior to submission.
+- **Data file formats:** DANDI only accepts data using standardized formats such
+  as [Neurodata Without Borders](https://nwb.org),[Brain Imaging Data Structure](https://bids.neuroimaging.io/),
+  [Neuroimaging Data Model](NIDM) and other [BRAIN Initiative](https://braininitiative.nih.gov/)
+  standards. We are working with the community to improve these standards and to
+  improve metadata to make the datasets more FAIR.
+- **Volume and size limitations:** There is a limit of 5TB per asset and we currently
+  accept any size of standardized datasets, as long as you can upload them over
+  an HTTP connection. However, if you plan to upload more than 50TB of data please
+  get in touch with us.
+- **Data quality:** All information is provided “as-is”, and the user shall hold
+  DANDI and information providers supplying data to Zenodo free and harmless in
+  connection with the use of such information.
+- **Metadata types and sources:** All metadata is stored internally in JSON-format
+  according to a defined JSON schema. Metadata is exported in several standard
+  formats such as JSON-LD and DataCite Metadata Schema.
+- **Language:** For textual items, English is preferred but all languages are
+  accepted.
+- **Licenses:** Users must specify a license for each dataset.
+
+## Access and Reuse
+
+- **Access to data objects:** Files may be deposited under open <!--or embargoed-->
+  access. Access to metadata and data files is provided over standard protocols
+  such as HTTP.
+- **Use and re-use of data objects:** Use and re-use is subject to the license
+  under which the data objects were deposited.
+- **Metadata access and reuse:** Metadata is licensed under CC0, except for email
+  addresses. All metadata is exported via HTTP and can be harvested.
```

Comment:
AINAL, but a bit tricky here
- factual "data" items (think - subject_id) might not even be copyrightable and thus not regulated by "License" terms. The same for email addresses -- alone, they aren't copyrightable AFAIK, thus we can't state that we have a license X for metadata but not for email addresses.
- collections of (meta)data, such as dataset and even NWB is a different topic
- we extract metadata for assets, which might be under non-CC0 license: we should somehow allow for such "derivative re-licensing".
- I would not mix in access protocols in here...

what about:

```suggestion
- **Metadata access and reuse:** Metadata records, provided by the users or extracted from the assets, are licensed under CC0. All metadata is made publicly available and can be harvested.
```